doc/conseil.sql

```diff
@@ -202,6 +213,14 @@ ALTER TABLE ONLY public.blocks
     ADD CONSTRAINT blocks_hash_key UNIQUE (hash);
 
 
+--
+-- Name: invalidated_blocks invalidated_blocks_pkey; Type: CONSTRAINT; Schema: public; Owner: -
+--
+
+ALTER TABLE ONLY public.invalidated_blocks
```

Comment:
What about a foreign-key from invalidated_blocks.hash to blocks.hash?

It would eliminate the possibility of a persisting the case numbered 4 on the documentation.

Will it be too restrictive in a way that I'm not seeing now?

Reply:
Not sure, will look into it, but not a bad idea.